aws-cpp-sdk-transfer/source/transfer/S3FileRequest.cpp

```diff
@@ -75,6 +75,11 @@ bool S3FileRequest::IsDone() const
 {
     return m_isDone.load();
 }
+    
+bool S3FileRequest::IsCancelled() const
```

Comment:
Cancel is handled in S3FileRequest::Cancel(), which calls UploadFileRequest::DoCancelAction, which then sends an AbortMultipartUploadRequest to the S3 client.